### PR TITLE
LEAF703 JSON scrubbed

### PR DESCRIPTION
**Pull request by jeremyrl**
ATO remediations

### CODE REVIEW

File: LEAF_Request_Portal/api/controllers/ConverterController.php [hunk 1]
```diff
@@ -42,7 +42,8 @@ public function post($act)
         });
 
         $this->index['POST']->register('converter/json', function ($args) {
-            return json_decode($_POST['input'], true);
+            trigger_error("done did the thing", E_USER_WARNING);
```
Comment (mgaoVA) [inline LEAF_Request_Portal/api/controllers/ConverterController.php]: Remove

File: LEAF_Request_Portal/sources/FormStack.php [hunk 2]
```diff
@@ -110,12 +110,12 @@ public function importForm($overwiteExisting = false)
         }
         if (isset($_POST['formPacket']))
         {
-            $formPacket = json_decode($_POST['formPacket'], true);
+            $formPacket = XSSHelpers::scrubObjectOrArray(json_decode($_POST['formPacket'], true));
```
Comment (mgaoVA) [inline LEAF_Request_Portal/sources/FormStack.php]: Need to revert this one and the one below it as it's necessary for functionality. Might need further discussion.

Comment (jeremyrl) [reply LEAF_Request_Portal/sources/FormStack.php]: Would it be better to do `sanitizeHTML` or `sanitizeHTMLRich`? Or do we need to try a different approach?

Comment (mgaoVA) [reply LEAF_Request_Portal/sources/FormStack.php]: Totally different approach. The issue is that importForm() is used as part of the Form Library system, where admins would be able to include and deploy custom HTML/JS.

**Comment by jeremyrl** [comment]
The requested changes have been made.

**Pull request merged**